app/controllers/application_controller.rb

```diff
@@ -1,7 +1,11 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery unless: -> { request.format.json? }, with: :exception
 
-  private
+  rescue_from VersionedFilesService::Lock::LockError do |e|
+    render build_error('423', e, 'Error acquiring lock')
+  end
+
+  protected
```

Comment:
Can you say more about choosing `protected` over `private` here?

Reply:
Adjusting so that not necessary.